analysis/db/max/README.md

```diff
@@ -0,0 +1,8 @@
+I think that the following questions might also be analyzed using available dataset, starting with titles and abstracts of the articles.
+
+-What sources, best practices or challenges are in medical care to prevent spreading
```

Comment:
how will you define a best practice or a challenge in the data pull?

Reply:
Maybe through the keywords like lack of smth (tests), invention or some other related words? It seems that it is not the most efficient way to do through keywords, but I haven't found another alternative yet.